test/metabase/query_processor_test/date_bucketing_test.clj

```diff
@@ -1244,34 +1244,37 @@
 ;; TODO -- is this really date BUCKETING? Does this BELONG HERE?!
 (deftest june-31st-test
   (testing "What happens when you try to add 3 months to March 31st? It should still work (#10072, #21968, #21969)"
-    ;; only testing the SQL drivers for now since I'm not 100% sure how to mock this for everyone else. Maybe one day
-    ;; when we support expressions like `+` for temporal types we can do an `:absolute-datetime` plus
-    ;; `:relative-datetime` expression and do this directly in MBQL.
-    (mt/test-drivers (filter #(isa? driver/hierarchy (driver/the-initialized-driver %) :sql)
+    ;; TODO Regex literals can't live under `mt/test-drivers` or any other macro with a clj-kondo hook or `:macroexpand`
+    ;; rule until https://github.com/clj-kondo/clj-kondo/issues/1987 is fixed and released. So this is bound in a let.
```

Comment:
Why don't we just keep `test-drivers` as a hook if it's such a problem? I think I was running into the same "unused value" issues you were and I fixed it like this in my branch

https://github.com/metabase/metabase/pull/28017/files#diff-42e5d16078af5746c40fe89161de9f1153463de76fcade9070051c0a7aac7cb7R43-R51

Reply:
Done as you suggested.